iorgen/parser_rust.py

```diff
@@ -68,7 +68,7 @@ def parse_without_error(type_: Type) -> bool:
 
     if type_.main == TypeEnum.LIST:
         assert type_.encapsulated is not None
-        return type_.encapsulated.main == TypeEnum.CHAR
+        return type_.encapsulated.main in (TypeEnum.CHAR, TypeEnum.STR)
```

Comment:
If we want to be conservative, we could write 
```suggestion
        return (type_.encapsulated.main == TypeEnum.CHAR
                or parse_without_error(type_.encapsulated))
```

Reply:
IMO the second suggestion is better since it is recursive and then provides support for more nesting, even if List[List[List[X]]] is silly, I believe it is better to do it this way.

I just fixed it so it is yapf-formated:
```suggestion
        return type_.encapsulated.main == TypeEnum.CHAR or parse_without_error(
            type_.encapsulated)
```

Reply:
Thanks for the fix!